code/modules/detectivework/forensics.dm

```diff
@@ -11,8 +11,10 @@ proc/is_complete_print(var/print)
 atom/var/list/suit_fibers
 
 atom/proc/add_fibers(mob/living/carbon/human/M)
+	var/obj/item/clothing/gloves/S = M.wear_suit
+	var/obj/item/clothing/gloves/U = M.w_uniform
+	var/obj/item/clothing/gloves/G = M.gloves
```

Comment:
Only cast these when necessary, so when you know they're wearing it

Reply:
Fixed 👍